Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlAcceleratorKeyPressedEventArgs.cs

```diff
@@ -0,0 +1,86 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Security;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
```

Comment:
@mcleanbyron All the types I took from WinRT to get our APIs to work without referencing the Windows.winmd file is in this namespace and folder. Documentation should be that of the type mentioned in `seealso` comment. Most places include a remark as to why there is a type that is duplicated with the same name from WinRT (though not this one, I'll clean that up). As I mentioned yesterday, open to renaming these so that there is a clear distinction between the types used by WebBrowser and WebViewControl, as we have plans for an adapter to bridge those two later.